src/__tests__/core/block.transactionsRoles.test.ts

```diff
@@ -236,8 +236,8 @@ test('block->transactionsRoles: both from and to addresses, no matching transact
 test('block->transactionsRoles: both from and to addresses, matching transactions returned', async () => {
   const query = `
     {
-      block(number: 5755715) {
-        transactionsRoles(from: "0xCcF1D27AfE45BA5F8c15265199E5b635AF4cb889", to: "0xA4eA687A2A7F29cF2dc66B39c68e4411C0D00C49") {
+      block(number: 5450945) {
+        transactionsRoles(from: "0x3f5CE5FBFe3E9af3971dD833D26bA9b5C936f0bE", to: "0x4156D3342D5c385a87D264F90653733592000581") {
```

Comment:
Is this an address that has sent multiple transactions on this block? If not, I'd change it to ensure that it's truly fetching the transaction with these specified 'from' and 'to' addresses.

Reply:
Good call!

Reply:
It has sent multiple transactions, the original test didn't which is why the bug slipped through :)

Reply:
Thanks for fixing @jonathanBuhler!! :)